cst438_gradebook/src/main/java/com/cst438/controllers/GradeBookController.java

```diff
@@ -161,24 +165,28 @@ public void updateGradebook (@RequestBody GradebookDTO gradebook, @PathVariable(
 		
 	}
 	
-	@GetMapping("/assignment/{id}")
-	@Transactional
-	public Assignment getAssignment(@PathVariable int assignment_id) {
-		Assignment assignment = assignmentRepository.findById(assignment_id).orElse(null);
-		
-		return assignment;
-	}
+//	@GetMapping("/assignment/{id}")
+//	@Transactional
+//	public Assignment getAssignment(@PathVariable int assignment_id) {
+//		Assignment assignment = assignmentRepository.findById(assignment_id).orElse(null);
+//		
+//		return assignment;
+//	}
```

Comment:
nit: looks like this is for debugging. May be should un-comment it for the future.